README.md

```diff
@@ -96,6 +101,12 @@ create schema public;
 
 You probably should be slightly less reckless than this.
 
+Run tests against Presto:
+
+```
+./run_tests.bash
```

Comment:
When I run this, I get:
```
Encountered an error:
Runtime Error
  No connection adapters were found for 'None://presto:8080/v1/statement'
```

I think this is related to setting `http_scheme=None` instead of `'http'`!

Reply:
It was related to the `http_scheme=None`. It's fixed now.